### DISCUSSION TITLE
macOS 14 (ARM), SANE, CanoScan LiDE 60: Scanner offline after first pass

### DISCUSSION THREAD

**Discussion by resoltico** [cyanfish/naps2#274] [Support]
_Hello !_

The issue I am reporting is that NAPS v7.2.2.0 does not perform a subsequent scan operation following a first pass after the creation of a new profile.

Please see the respective video recording of this problem at the link below:

https://vimeo.com/902167956/69b372b3ba

_Thank you very much !_

**Comment by cyanfish** [comment]
It looks like the genesys backend changes its ID after a scan, which prevents NAPS2 from re-using it. The best thing I can recommend for now is to not specify a device in your profile. Then it will prompt to choose a device each time you scan, which should work.
